redis/src/test/scala/zio/redis/SecondRedisExecutor.scala

```diff
@@ -0,0 +1,24 @@
+package zio.redis
+
+import zio.Has
+import zio.ZLayer
+import zio.Chunk
+import zio.logging.Logging
+
+object SecondRedisExecutorLayer {
```

Comment:
This is the main problem area. Trying to figure out how to make a new type of Redis Executor. Ideally, it would just reuse RedisExecutor but the class is quite complex and I'm having a hard time making it happen. @adamgfraser suggested using a case class, but not sure how. Perhaps as a simple wrapper? But then I'm not sure how to make that a ZLayer that can be threaded through the test.